### PR TITLE
Prepare for community solutions

### PR DESCRIPTION
**Pull request by iHiD**

### CODE REVIEW

File: app/commands/solution/search_community_solutions.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Solution
+  class SearchCommunitySolutions
+    include Mandate
+
+    DEFAULT_PAGE = 1
+    DEFAULT_PER = 25
```
Comment (ErikSchierboom) [inline app/commands/solution/search_community_solutions.rb:6]: Was there a reason not to make these private?

File: app/models/solution.rb [hunk 2]
```diff
@@ -120,19 +120,24 @@ def update_mentoring_status!
...
-  # TODO: Erik
+  # TODO
+  def num_views
+    1270
+  end
+
+  # TODO
   def num_loc
-    9
+    18
```
Comment (ErikSchierboom) [inline app/models/solution.rb:130]: Just a side note: how do we want to implement this? Using [cloc](https://github.com/AlDanial/cloc) or should tracks implement this themselves? Or do we just have line count?

**Review by ErikSchierboom — APPROVED** [review]

File: app/commands/solution/search_community_solutions.rb (hunk 1 above)
Comment (iHiD) [reply app/commands/solution/search_community_solutions.rb:6]: Now: I copied this from the Searching of User Solutions.
Previously: 🤷 No, probably not!

File: app/models/solution.rb (hunk 2 above)
Comment (iHiD) [reply app/models/solution.rb:130]: Yeah, `cloc` running via Lambda could work!

Comment (iHiD) [reply app/models/solution.rb:130]: https://github.com/exercism/website/issues/668

**Pull request merged**